### ISSUE TITLE
Add list of texlive packages

### ISSUE THREAD

**Issue by pharpend** [learnmath/lysa#19]
We have a list of required TeX packages in the travis file. Let's put them in the readme and the contributing guide?

**Comment by pharpend** [comment]
Maybe let's do this for every OS. I'll write the Gentoo section. Everyone write the section for their OS.

**Comment by DTSCode** [comment]
Ok... I'll add Ubuntu tonite. Does this change at all in terms of #20?

-----Original Message-----

**Comment by wei2912** [comment]
Ubuntu's list can be found in `.travis.yml`.

```
sudo apt-get install biblatex texlive texlive-latex-extra texlive-bibtex-extra texlive-xetex texlive-math-extra texlive-latex3
```

I will work on the list for Fedora.

**Comment by daniel-j-h** [comment]
> Does this change at all in terms of #20?

Yes, latexmk is required in addition for this branch.

**Comment by pharpend** [comment]
I agree with @daniel-j-h .

I would like to see this issue fixed before we merge #20, merely so that @daniel-j-h can include the required latex packages for latex-mk (assuming they are different). I don't know, I've never used latex-mk.

**Comment by ghost** [comment]
@daniel-j-h @pharpend agreed.

**Comment by pharpend** [comment]
I'll wait until @beingbrown's thing is merged, then add the Gentoo part to the contrib guide. Then, I'll assert that this is "fixed", and merge #20. Sound good?

**Comment by pharpend** [comment]
I ended up rejecting #20, so this looks to be its own issue. I'll fix it in a minute.

**Comment by ghost** [comment]
Is this done now?

**Comment by ghost** [comment]
Sorry, went to click to assign and got 'assign yourself' instead.

**Comment by pharpend** [comment]
Yes, I fixed-ish this in #39. I added instructions for Gentoo and Ubuntu, but I imagine there might be people who use other operating systems.
